src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1399,7 +1399,7 @@ internal enum ErrorCode
         ERR_RefReturnNonreturnableLocal = 8158,
         ERR_RefReturnNonreturnableLocal2 = 8159,
         ERR_RefReturnRangeVariable = 8160,
-        ERR_RefReturnRangeVariable2 = 8170,
+        // Available  = 8161-8170
```

Comment:
I didn't realize I had left a gap of 10 here when compacting. Consider compacting more if you want.